Added options to ignore order by in deletes and updates (#89)
singlestore-labs/singlestoredb-laravel-driver: README.md

]) : [],
```

> ## Order by in delete and update

Reviewer: ```suggestion
## `ORDER BY` Clause in `DELETE` and `UPDATE` Queries
```

> SingleStore doesn't support the ORDER BY clause in DELETE and UPDATE queries.

Reviewer: ```suggestion
SingleStore does not support the `ORDER BY` clause in the `DELETE` and `UPDATE` queries.
```

> Issuing queries like the following will result in an error. 

Reviewer: ```suggestion
Issuing queries similar to the following will return an error. 
```

```php
DB::table('test')->orderBy('id', 'asc')->update(['id' => 1, 'a' => 'b']);
DB::table('test')->orderBy('id', 'asc')->delete();
```

> You can configure the driver to ignore `orderBy` in such requests using `ignore_order_by_in_deletes` and

Reviewer: ```suggestion
You can configure the driver to ignore `orderBy` in `delete()` and `update()` requests by enabling `ignore_order_by_in_deletes` and
```

> `ignore_order_by_in_updates` configurations. For example:

Reviewer: ```suggestion
`ignore_order_by_in_updates` in the connection configuration, respectively. For example:
```

```php
[
    'default' => env('DB_CONNECTION', 'singlestore'),

    'connections' => [
        'singlestore' => [
            'driver' => 'singlestore',
            'ignore_order_by_in_deletes' => true,
            'ignore_order_by_in_updates' => true,
            ...
        ],
    ]
]
```

> Note that when `orderBy` is ignored it may result in deletion/update of different rows.

Reviewer: ```suggestion
Note that when `orderBy` is ignored, it may result in deletion/update of different rows.
```

> Note that when `orderBy` is ignored, it may result in deletion/update of different rows.

Reviewer 2: Is it possible to be more specific about when this can happen? Is presence of LIMIT/OFFSET the only factor or not?

Author: Yep, it is the only factor
